app/src/main/java/com/boosters/promise/ui/invite/InviteActivity.kt

```diff
@@ -1,46 +1,106 @@
 package com.boosters.promise.ui.invite
 
+import android.content.Intent
+import android.os.Build
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import androidx.activity.viewModels
 import androidx.databinding.DataBindingUtil
 import com.boosters.promise.R
 import com.boosters.promise.databinding.ActivityInviteBinding
+import com.boosters.promise.ui.invite.adapter.FriendAdapter
+import com.boosters.promise.ui.invite.adapter.MemberAdapter
+import com.boosters.promise.ui.invite.model.UserUiState
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
 class InviteActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityInviteBinding
     private val inviteViewModel: InviteViewModel by viewModels()
-    private val inviteAdapter = InviteAdapter()
+    private val friendAdapter = FriendAdapter()
+    private val memberAdapter = MemberAdapter()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = DataBindingUtil.setContentView(this, R.layout.activity_invite)
         setBinding()
-        observeAllFriendItems()
         observeCurrentFriendItems()
+        observeCurrentMemberItems()
+        setFriendItemClickListener()
+        setMemberItemClickListener()
+        setConfirmButtonClickListener()
 
-        inviteViewModel.loadAllFriendItems()
+        inviteViewModel.initAllFriendItems()
+        initMemberItems()
     }
 
     private fun setBinding() {
         binding.lifecycleOwner = this
         binding.viewModel = inviteViewModel
-        binding.recyclerViewInviteFriendList.adapter = inviteAdapter
+        binding.recyclerViewInviteFriendList.adapter = friendAdapter
+        binding.recyclerViewInviteMemberList.adapter = memberAdapter
     }
 
-    private fun observeAllFriendItems() {
-        inviteViewModel.allFriendItems.observe(this) { friendItems ->
-            inviteViewModel.setCurrentFriendItems(friendItems)
+    private fun observeCurrentFriendItems() {
+        inviteViewModel.currentFriendItems.observe(this) { friendItems ->
+            friendAdapter.submitList(friendItems)
         }
     }
 
-    private fun observeCurrentFriendItems() {
-        inviteViewModel.currentFriendItems.observe(this) { friendItems ->
-            inviteAdapter.submitList(friendItems)
+    private fun observeCurrentMemberItems() {
+        inviteViewModel.currentMemberItems.observe(this) { memberItems ->
+            memberAdapter.submitList(memberItems)
         }
     }
 
+    private fun setFriendItemClickListener() {
+        friendAdapter.setOnItemClickListener(object : FriendAdapter.OnItemClickListener {
+            override fun onItemClick(user: UserUiState, pos: Int) {
+                if (user !in memberAdapter.currentList) {
```

Comment:
부정 연산자보다는 `not()`을 사용하는 것도 고려해봐주세요.
```suggestion
                if (memberAdapter.currentList.contains(user).not()) {
```